README.md

```diff
@@ -3,3 +3,42 @@
 Pulumi components carefully crafted by Nuage
 
 TODO: use [the official repo](https://github.com/pulumi/pulumi-component-provider-py-boilerplate) to build a Pulumi Component Provider
+
+# Install the Pulumi Plugin
+
+To use this resource, you need to install Pulumi plugin first (independent from the development language).
+
+1. Download the latest release from https://github.com/nuage-studio/pulumi-nuage/releases/
+2. Install the .tar.gz package using Pulumi
+```
+pulumi plugin install resource nuage 0.0.1 --file pulumi-resource-nuage-v0.0.1-darwin-amd64.tar.gz
+```
+
+You may also automate this process via Taskfile. Here is an example:
+```
+version: "3"
+
+tasks:
+  pulumi_install:
+    desc: Downloads binary
+    env:
+      FILE_NAME: "pulumi-resource-nuage-v0.0.1-darwin-amd64.tar.gz"
+    cmds:
+      - curl -L -o $FILE_NAME https://github.com/nuage-studio/pulumi-nuage/releases/download/0.0.1/${FILE_NAME}
+      - pulumi plugin install resource nuage 0.0.1 --file $FILE_NAME
+      - rm $FILE_NAME
+```
+
+# Setup Instructions for Python SDK
+
+* You can add following line under the `[tool.poetry.dependencies]`, to use Python package.
+```
+pulumi-nuage = { git = "https://github.com/nuage-studio/pulumi-nuage.git", subdirectory = "sdk/python" }
+```
+* Then you need to run `poetry install`
+
+
+# Start Working with us
+You can configure your Nuage SSO with the pre-prepared Cloudformation template using the following link:
```

Comment:
"
If you want to collaborate with us, feel free to contact us at contact@nuage.studio

Should you need to give us access to your existing AWS account, you can use the Cloudformation link below which will deploy an AWS IdentityProvider that trusts our SSO.

You will need a SAML Metadata document (XML) which we'll happily provide you via email.
"